### ISSUE TITLE
Dropdown list issues

### ISSUE THREAD

**Issue by Aussiemon** [Darktide-Mod-Framework/Darktide-Mod-Framework#12]
* Selecting a dropdown entry above a keybind will prompt for a keypress
* Dropdown entries scroll from the top instead of the bottom? It's weird.

**Comment by Aussiemon** [comment]
https://github.com/Darktide-Mod-Framework/Darktide-Mod-Framework/commit/99799a4a10d9bf1e6656f991faf0db15f0dcda14

**Comment by deluxghost** [comment]
I found this in crosshair remap mod, guess it's an issue of dmf?

if the dropdown shows above the text part, options are reversed

says there are options ABCDE, normally it is like:

```
[ Option    v ]    [ Option    v ]
 A                  B
 B  --> scroll -->  C
 C                  D
```

but when the dropdown expands up:

```
 C                  D
 B  --> scroll -->  C
 A                  B
[ Option    v ]    [ Option    v ]
```

the options shown are still correct, but reversed, which is annoyed

**Comment by Aussiemon** [comment]
Should be addressed by [3b387c294d746f2f856379e8569bf12169a6443c](https://github.com/Darktide-Mod-Framework/Darktide-Mod-Framework/commit/3b387c294d746f2f856379e8569bf12169a6443c)
